src/openms/include/OpenMS/DATASTRUCTURES/DateTime.h

```diff
@@ -66,17 +67,20 @@ namespace OpenMS
     DateTime();
 
     /// Copy constructor
-    DateTime(const DateTime& date) = default;
+    DateTime(const DateTime& date);
```

Comment:
can this be noexcept?

Reply:
technically no, since it allocates.

Reply:
where is the advantage of noexcept here?
Its a copy anyway, not a move.

Reply:
probably none... I thought it might have a slight performance impact but don't know for sure

Reply:
mhh.. adding `noexcept` might degrade performance actually. If the compiler cannot prove that no exception can be thrown, then it needs to wrap the function into a try-catch-block which ensures that terminate will be called upon throwing an exception. That is not cheap...